packages/common/src/util.ts

```diff
@@ -309,3 +309,19 @@ export function extend(
     return obj;
   }, extended);
 }
+
+/**
+ * Add number of days to a given date
+ *
+ * @export
+ * @param {string} date
+ * @param {number} numOfDays
+ * @returns {string} date string with numOfDays added to the given date
+ */
+export function addDays(date: string, numOfDays: number): string {
+  const given = new Date(date);
```

Comment:
Is there error handling in place to cover the case where `new Date(date)` returns `Invalid Date`?

Reply:
Yes, It is handled by hub-indexer here - https://github.com/ArcGIS/hub-indexer/blob/master/packages/api/search/items.js#L27-L32

Reply:
It's not going to get that far:  

```
> addDays('this is not a date', 1)
RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at addDays (repl:5:5)
```